lstein traceback and exit with error code 255

<!--  PLEASE FILL THIS OUT, IT WILL MAKE BOTH OF OUR LIVES EASIER -->

**Has this issue been opened before?**

- [✓] It is not in the [FAQ](https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/FAQ), I checked.
- [✓] It is not in the [issues](https://github.com/AbdBarho/stable-diffusion-webui-docker/issues?q=), I searched.


**Describe the bug**

Trackback errors and exit code 255 when trying to run lstein

**Which UI**

Specific to lstein. Auto and hlky both function normally.


**Hardware / Software**
 - OS: Arch Linux
 - OS version: [rolling]
 - Docker Version: 20.10.21
 - Docker compose version: 2.13.0
 - Repo version: 3.2.0-3-gf1a1641
 - RAM: 16 GB
 - GPU/VRAM: 8 GB

**Steps to Reproduce**
1. Clean install of arch linux with nvidia 3080 and packages: nvidia-container-toolkit, nvidia, cuda, docker, docker-compose
2. Clone repo
3. docker compose --profile download up --build
4. docker compose --profile lstein up --build
5. See error below

**Additional context**
```sh
[+] Building 1.3s (20/20) FINISHED                                                                                                                                             
 => [internal] load build definition from Dockerfile                                                                                                                      0.0s
 => => transferring dockerfile: 32B                                                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                                                         0.0s
 => => transferring context: 2B                                                                                                                                           0.0s
 => resolve image config for docker.io/docker/dockerfile:1                                                                                                                0.6s
 => CACHED docker-image://docker.io/docker/dockerfile:1@sha256:9ba7531bd80fb0a858632727cf7a112fbfd19b17e94c4e84ced81e24ef1a0dbc                                           0.0s
 => [internal] load .dockerignore                                                                                                                                         0.0s
 => [internal] load build definition from Dockerfile                                                                                                                      0.0s
 => [internal] load metadata for docker.io/library/python:3.10-slim                                                                                                       0.3s
 => [ 1/11] FROM docker.io/library/python:3.10-slim@sha256:030ead045da5758362ae198e9025671f22490467312dbad9af6b29a6d6bc029b                                               0.0s
 => [internal] load build context                                                                                                                                         0.0s
 => => transferring context: 123B                                                                                                                                         0.0s
 => CACHED [ 2/11] RUN pip install torch==1.12.0+cu116 --extra-index-url https://download.pytorch.org/whl/cu116                                                           0.0s
 => CACHED [ 3/11] RUN apt-get update && apt-get install git -y && apt-get clean                                                                                          0.0s
 => CACHED [ 4/11] RUN git clone https://github.com/invoke-ai/InvokeAI.git /stable-diffusion                                                                              0.0s
 => CACHED [ 5/11] WORKDIR /stable-diffusion                                                                                                                              0.0s
 => CACHED [ 6/11] RUN <<EOF (git reset --hard 5c31feb3a1096d437c94b6e1c3224eb7a7224a85...)                                                                               0.0s
 => CACHED [ 7/11] RUN <<EOF (apt-get update...)                                                                                                                          0.0s
 => CACHED [ 8/11] RUN <<EOF (git fetch...)                                                                                                                               0.0s
 => CACHED [ 9/11] RUN pip install --force-reinstall opencv-python-headless && python3 -c "from patchmatch import patch_match"                                            0.0s
 => CACHED [10/11] COPY . /docker/                                                                                                                                        0.0s
 => CACHED [11/11] RUN <<EOF (python3 /docker/info.py /stable-diffusion/frontend/dist/index.html...)                                                                      0.0s
 => exporting to image                                                                                                                                                    0.1s
 => => exporting layers                                                                                                                                                   0.0s
 => => writing image sha256:5c77c04a1d392f80c2313a5e1acb578e0cdbbbfaede5d83e398f1847620555a6                                                                              0.0s
 => => naming to docker.io/library/sd-lstein:9                                                                                                                            0.0s
[+] Running 1/0
 ⠿ Container webui-docker-lstein-1  Created                                                                                                                               0.0s
Attaching to webui-docker-lstein-1
webui-docker-lstein-1  | Mounted .cache
webui-docker-lstein-1  | Mounted transformers
webui-docker-lstein-1  | Mounted clipseg
webui-docker-lstein-1  | Mounted .cache
webui-docker-lstein-1  | Mounted RealESRGAN
webui-docker-lstein-1  | Mounted transformers
webui-docker-lstein-1  | Mounted GFPGANv1.4.pth
webui-docker-lstein-1  | Mounted Codeformer
webui-docker-lstein-1  | Mounted transformers
webui-docker-lstein-1  | Loading Python libraries...
webui-docker-lstein-1  | 
webui-docker-lstein-1  | Installing bert tokenizer (ignore deprecation errors)...Welcome to InvokeAI. This script will help download the Stable Diffusion weight files
webui-docker-lstein-1  | and other large models that are needed for text to image generation. At any point you may interrupt
webui-docker-lstein-1  | this program and resume later.
webui-docker-lstein-1  | 
webui-docker-lstein-1  | 
webui-docker-lstein-1  | ** DOWNLOADING SUPPORT MODELS **
webui-docker-lstein-1  | 
webui-docker-lstein-1  | 
webui-docker-lstein-1  | A problem occurred during initialization.
webui-docker-lstein-1  | The error was: "[Errno 2] No such file or directory: '/stable-diffusion/models/bert-base-uncased/models--bert-base-uncased'"
webui-docker-lstein-1  | Traceback (most recent call last):
webui-docker-lstein-1  |   File "/stable-diffusion/scripts/configure_invokeai.py", line 746, in main
webui-docker-lstein-1  |     download_bert()
webui-docker-lstein-1  |   File "/stable-diffusion/scripts/configure_invokeai.py", line 419, in download_bert
webui-docker-lstein-1  |     download_from_hf(BertTokenizerFast,'bert-base-uncased')
webui-docker-lstein-1  |   File "/stable-diffusion/scripts/configure_invokeai.py", line 425, in download_from_hf
webui-docker-lstein-1  |     return model_class.from_pretrained(model_name,
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/site-packages/transformers/tokenization_utils_base.py", line 1700, in from_pretrained
webui-docker-lstein-1  |     resolved_config_file = cached_file(
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/site-packages/transformers/utils/hub.py", line 409, in cached_file
webui-docker-lstein-1  |     resolved_file = hf_hub_download(
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/site-packages/huggingface_hub/file_download.py", line 1024, in hf_hub_download
webui-docker-lstein-1  |     os.makedirs(storage_folder, exist_ok=True)
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/os.py", line 225, in makedirs
webui-docker-lstein-1  |     mkdir(name, mode)
webui-docker-lstein-1  | FileNotFoundError: [Errno 2] No such file or directory: '/stable-diffusion/models/bert-base-uncased/models--bert-base-uncased'
webui-docker-lstein-1  | 
webui-docker-lstein-1  | + python3 -u scripts/invoke.py --web --host 0.0.0.0 --port 7860 --config /docker/models.yaml --root_dir /stable-diffusion --outdir /output
webui-docker-lstein-1  | * Initializing, be patient...
webui-docker-lstein-1  | >> Initialization file /root/.invokeai found. Loading...
webui-docker-lstein-1  | >> InvokeAI runtime directory is "/stable-diffusion"
webui-docker-lstein-1  | >> Patchmatch initialized
webui-docker-lstein-1  | >> GFPGAN Initialized
webui-docker-lstein-1  | ## NOT FOUND: CodeFormer model not found at /stable-diffusion/models/codeformer/codeformer.pth
webui-docker-lstein-1  | >> CodeFormer Disabled
webui-docker-lstein-1  | >> ESRGAN Initialized
webui-docker-lstein-1  | >> Using device_type cuda
webui-docker-lstein-1  | >> Current VRAM usage:  0.00G
webui-docker-lstein-1  | >> Scanning Model: stable-diffusion-1.5
webui-docker-lstein-1  | >> Model Scanned. OK!!
webui-docker-lstein-1  | >> Loading stable-diffusion-1.5 from /data/StableDiffusion/v1-5-pruned-emaonly.ckpt
webui-docker-lstein-1  | >> Calculating sha256 hash of weights file
webui-docker-lstein-1  | >> sha256 = cc6cb27103417325ff94f52b7a5d2dde45a7515b25c255d8e396c90014281516 (7.14s)
webui-docker-lstein-1  |    | LatentDiffusion: Running in eps-prediction mode
webui-docker-lstein-1  |    | DiffusionWrapper has 859.52 M params.
webui-docker-lstein-1  |    | Making attention of type 'vanilla' with 512 in_channels
webui-docker-lstein-1  |    | Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
webui-docker-lstein-1  |    | Making attention of type 'vanilla' with 512 in_channels
webui-docker-lstein-1  | ** model stable-diffusion-1.5 could not be loaded: [Errno 2] No such file or directory: '/stable-diffusion/models/openai/clip-vit-large-patch14/models--openai--clip-vit-large-patch14'
webui-docker-lstein-1  | Traceback (most recent call last):
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/invoke/model_cache.py", line 80, in get_model
webui-docker-lstein-1  |     requested_model, width, height, hash = self._load_model(model_name)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/invoke/model_cache.py", line 231, in _load_model
webui-docker-lstein-1  |     model = instantiate_from_config(omega_config.model)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/util.py", line 90, in instantiate_from_config
webui-docker-lstein-1  |     return get_obj_from_str(config['target'])(
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/models/diffusion/ddpm.py", line 658, in __init__
webui-docker-lstein-1  |     self.instantiate_cond_stage(cond_stage_config)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/models/diffusion/ddpm.py", line 769, in instantiate_cond_stage
webui-docker-lstein-1  |     model = instantiate_from_config(config)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/util.py", line 90, in instantiate_from_config
webui-docker-lstein-1  |     return get_obj_from_str(config['target'])(
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/modules/encoders/modules.py", line 247, in __init__
webui-docker-lstein-1  |     self.tokenizer = CLIPTokenizer.from_pretrained(
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/site-packages/transformers/tokenization_utils_base.py", line 1736, in from_pretrained
webui-docker-lstein-1  |     resolved_vocab_files[file_id] = cached_file(
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/site-packages/transformers/utils/hub.py", line 409, in cached_file
webui-docker-lstein-1  |     resolved_file = hf_hub_download(
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/site-packages/huggingface_hub/file_download.py", line 1024, in hf_hub_download
webui-docker-lstein-1  |     os.makedirs(storage_folder, exist_ok=True)
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/os.py", line 225, in makedirs
webui-docker-lstein-1  |     mkdir(name, mode)
webui-docker-lstein-1  | FileNotFoundError: [Errno 2] No such file or directory: '/stable-diffusion/models/openai/clip-vit-large-patch14/models--openai--clip-vit-large-patch14'
webui-docker-lstein-1  | 
webui-docker-lstein-1  | 
webui-docker-lstein-1  | !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
webui-docker-lstein-1  |    You appear to have a missing or misconfigured model file(s).                   
webui-docker-lstein-1  |    The script will now exit and run configure_invokeai.py to help fix the problem.
webui-docker-lstein-1  |    After reconfiguration is done, please relaunch invoke.py.                      
webui-docker-lstein-1  | !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
webui-docker-lstein-1  | configure_invokeai is launching....
webui-docker-lstein-1  | 
webui-docker-lstein-1  | Loading Python libraries...
webui-docker-lstein-1  | 
webui-docker-lstein-1  | Welcome to InvokeAI. This script will help download the Stable Diffusion weight files
webui-docker-lstein-1  | and other large models that are needed for text to image generation. At any point you may interrupt
webui-docker-lstein-1  | this program and resume later.
webui-docker-lstein-1  | 
webui-docker-lstein-1  | ** INITIALIZING INVOKEAI RUNTIME DIRECTORY **
webui-docker-lstein-1  | Select a directory in which to install InvokeAI's models and configuration files [/root/invokeai]: 
webui-docker-lstein-1  | A problem occurred during initialization.
webui-docker-lstein-1  | The error was: "EOF when reading a line"
webui-docker-lstein-1  | Traceback (most recent call last):
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/invoke/model_cache.py", line 80, in get_model
webui-docker-lstein-1  |     requested_model, width, height, hash = self._load_model(model_name)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/invoke/model_cache.py", line 231, in _load_model
webui-docker-lstein-1  |     model = instantiate_from_config(omega_config.model)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/util.py", line 90, in instantiate_from_config
webui-docker-lstein-1  |     return get_obj_from_str(config['target'])(
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/models/diffusion/ddpm.py", line 658, in __init__
webui-docker-lstein-1  |     self.instantiate_cond_stage(cond_stage_config)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/models/diffusion/ddpm.py", line 769, in instantiate_cond_stage
webui-docker-lstein-1  |     model = instantiate_from_config(config)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/util.py", line 90, in instantiate_from_config
webui-docker-lstein-1  |     return get_obj_from_str(config['target'])(
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/modules/encoders/modules.py", line 247, in __init__
webui-docker-lstein-1  |     self.tokenizer = CLIPTokenizer.from_pretrained(
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/site-packages/transformers/tokenization_utils_base.py", line 1736, in from_pretrained
webui-docker-lstein-1  |     resolved_vocab_files[file_id] = cached_file(
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/site-packages/transformers/utils/hub.py", line 409, in cached_file
webui-docker-lstein-1  |     resolved_file = hf_hub_download(
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/site-packages/huggingface_hub/file_download.py", line 1024, in hf_hub_download
webui-docker-lstein-1  |     os.makedirs(storage_folder, exist_ok=True)
webui-docker-lstein-1  |   File "/usr/local/lib/python3.10/os.py", line 225, in makedirs
webui-docker-lstein-1  |     mkdir(name, mode)
webui-docker-lstein-1  | FileNotFoundError: [Errno 2] No such file or directory: '/stable-diffusion/models/openai/clip-vit-large-patch14/models--openai--clip-vit-large-patch14'
webui-docker-lstein-1  | 
webui-docker-lstein-1  | During handling of the above exception, another exception occurred:
webui-docker-lstein-1  | 
webui-docker-lstein-1  | Traceback (most recent call last):
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/invoke/CLI.py", line 115, in main
webui-docker-lstein-1  |     gen.load_model()
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/generate.py", line 818, in load_model
webui-docker-lstein-1  |     self.set_model(self.model_name)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/generate.py", line 843, in set_model
webui-docker-lstein-1  |     model_data = cache.get_model(model_name)
webui-docker-lstein-1  |   File "/stable-diffusion/ldm/invoke/model_cache.py", line 91, in get_model
webui-docker-lstein-1  |     assert self.current_model,'** FATAL: no current model to restore to'
webui-docker-lstein-1  | AssertionError: ** FATAL: no current model to restore to
webui-docker-lstein-1  | 
webui-docker-lstein-1  | During handling of the above exception, another exception occurred:
webui-docker-lstein-1  | 
webui-docker-lstein-1  | Traceback (most recent call last):
webui-docker-lstein-1  |   File "/stable-diffusion/scripts/configure_invokeai.py", line 740, in main
webui-docker-lstein-1  |     initialize_rootdir(Globals.root,opt.yes_to_all)
webui-docker-lstein-1  |   File "/stable-diffusion/scripts/configure_invokeai.py", line 608, in initialize_rootdir
webui-docker-lstein-1  |     root = select_root(root,yes_to_all)
webui-docker-lstein-1  |   File "/stable-diffusion/scripts/configure_invokeai.py", line 587, in select_root
webui-docker-lstein-1  |     directory = input(f"Select a directory in which to install InvokeAI's models and configuration files [{default}]: ").strip(' \\')
webui-docker-lstein-1  | EOFError: EOF when reading a line
webui-docker-lstein-1  | 
webui-docker-lstein-1 exited with code 255```

--- a/services/lstein/entrypoint.sh
+++ b/services/lstein/entrypoint.sh
@@ -5,7 +5,7 @@ set -Eeuo pipefail
 declare -A MOUNTS
 
 # cache
-MOUNTS["/root/.cache"]=/data/.cache
+MOUNTS["/root/.cache"]=/data/.cache/
 
 # ui specific
 MOUNTS["${ROOT}/models/codeformer"]=/data/Codeformer/
@@ -15,9 +15,9 @@ MOUNTS["${ROOT}/models/gfpgan/weights"]=/data/.cache/
 
 MOUNTS["${ROOT}/models/realesrgan"]=/data/RealESRGAN/
 
-MOUNTS["${ROOT}/models/bert-base-uncased"]=/data/.cache/huggingface/transformers
-MOUNTS["${ROOT}/models/openai/clip-vit-large-patch14"]=/data/.cache/huggingface/transformers
-MOUNTS["${ROOT}/models/CompVis/stable-diffusion-safety-checker"]=/data/.cache/huggingface/transformers
+MOUNTS["${ROOT}/models/bert-base-uncased"]=/data/.cache/huggingface/transformers/
+MOUNTS["${ROOT}/models/openai/clip-vit-large-patch14"]=/data/.cache/huggingface/transformers/
+MOUNTS["${ROOT}/models/CompVis/stable-diffusion-safety-checker"]=/data/.cache/huggingface/transformers/
 
 
 # hacks

